src/framework/dropdown/_dropdownOption.scss

```diff
@@ -9,4 +9,8 @@
   &.is-dropdown-option-focus {
     background-color: #f3f9fd;
   }
+
+  &.dropdownGroup--first {
+    border-top: 1px dashed rgba(122, 129, 131, 0.4);
```

Comment:
`1px dashed rgba(#7a8183, 0.4)` would also do. Or even a predefined color (as of not introducing too many different colors even though mockup does). Up to you...

Reply:
`z-index` should be adjusted so that the line is drawn below the scrollbar and not over it.

Reply:
scrollbar is transparent